cmake/platform.cmake

```diff
@@ -233,6 +241,9 @@ elseif(UNIX OR MINGW)
         append(CMAKE_CCXX_FLAGS "-Wsign-compare")
     endif()
 
+    # Generating frame pointers for easier performance profiling
+    append(CMAKE_CCXX_FLAGS "-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer")
```

Comment:
Did you run any performance measurements? I don't think we should see any issue (most of the performance is in jitted kernels), but it might be worth checking to be on the safe side.

Reply:
The perf might also be arch dependant in this case.

Reply:
This is for Windows and Linux, I am curious to know the perf effect on Linux on AArch64. We might need to do a quick sanity check.

Reply:
There is no visible performance difference in execution time on JIT version, but weirdly enough kernel creation is consistently faster with frame pointer enabled version by ax 5%. In REF implementation both kernel creation and execution time fluctuates by ax 5% when using full socket per instance but on smaller 4-core per instance limited runs the performance is identical.

Reply:
@spalicki I'll run similar benchmarking for aarch64 (I assume your results are x64-only?) and get back to you shortly. Would appreciate it if you could hold off from merging in the meantime.

Reply:
@spalicki @mgouicem Thanks for your patience. I've tested this now and it looks like it does cause a noticeable performance drop for me on Neoverse (R) V1.  I used the matmul tests as a benchmark. There seem to be a slowdown against all tests but as an example:

```
./base/tests/benchdnn/benchdnn --mode=P --matmul --perf-template="%impl%,%name%,%prb%,%-ctime%,%-time%" --dt=bf16:bf16:bf16 --attr-post-ops=linear:2:1 2048x256:256x128_n"NCF:1*1"  
Output template: %impl%,%name%,%prb%,%-ctime%,%-time%
gemm:jit:bf16,NCF:1*1,--mode=P --matmul --dt=bf16:bf16:bf16 --attr-post-ops=linear:2:1 2048x256:256x128_nNCF:1*1,0.0136719,2.42505
tests:1 passed:1 skipped:0 mistrusted:0 unimplemented:0 invalid_arguments:0 failed:0 listed:0
total perf: min(ms):2.42505 avg(ms):2.43697
total: 3.02s; fill: 0.01s (0%);

./pr2244/tests/benchdnn/benchdnn --mode=P --matmul --perf-template="%impl%,%name%,%prb%,%-ctime%,%-time%" --dt=bf16:bf16:bf16 --attr-post-ops=linear:2:1 2048x256:256x128_n"NCF:1*1"
Output template: %impl%,%name%,%prb%,%-ctime%,%-time%
gemm:jit:bf16,NCF:1*1,--mode=P --matmul --dt=bf16:bf16:bf16 --attr-post-ops=linear:2:1 2048x256:256x128_nNCF:1*1,0.0146484,3.62427
tests:1 passed:1 skipped:0 mistrusted:0 unimplemented:0 invalid_arguments:0 failed:0 listed:0
total perf: min(ms):3.62427 avg(ms):3.63837
total: 3.02s; fill: 0.01s (0%);
```

Let's skip this on aarch64 for now. Thanks.